lib/typescript/httpclient/endpoints/readConversations.ts

```diff
@@ -0,0 +1,7 @@
+import {doFetchFromBackend} from '../api';
+
+export function readConversations(conversationId: string) {
+  return doFetchFromBackend('conversations.read', {conversation_id: conversationId})
+    .then(() => Promise.resolve(true))
+    .catch((error: Error) => Promise.reject(error));
```

Comment:
The `catch` line is redundant because it catches and immediately throws the error again